apps/css-workshop/src/pages/anchor.astro

```diff
@@ -15,24 +16,37 @@ import Layout from './_layout.astro';
 
   <section id='demo-default'>
     <p>
-      <a href='https://itwin.github.io/iTwinUI/' class='iui-anchor' id='test-1'
-        >iTwinUI components list</a
-      > - A basic anchor link.
+      <Anchor_ href='https://itwin.github.io/iTwinUI/' id='test-1'>iTwinUI components list</Anchor_>
+       - A basic anchor link.
     </p>
 
     <p>
-      <a
-        href='https://www.bentley.com/en'
-        class='iui-anchor iui-anchor-external'
-        referrerpolicy='no-referrer'
-        target='_blank'
-        id='test-2'>Bentley Systems<span class='iui-visually-hidden'>(opens in new tab)</span></a
-      > - An anchor link that opens in a new tab / window.
+      <Anchor_ external='true' href='https://www.bentley.com/en'>
+        Bentley Systems<span class='iui-visually-hidden'>(opens in new tab)</span>
+      </Anchor_>
```

Comment:
This is the only test image that is still failing and I am not sure how to solve it. The white space is being added before the pseudo element.
![anchor-dif](https://github.com/iTwin/iTwinUI/assets/849817/e2600d33-d771-4fd7-8a0d-ac975fabc14f)

Reply:
What if you remove whitespace from start and end?

```jsx
<Anchor_ …>…</Anchor_>
```
rather than

```jsx
<Anchor_ …>
  …
</Anchor_>
```

Reply:
I've got this working:
```html
<!-- prettier-ignore -->
<Anchor_
  …
>Bentley Systems<span class='iui-visually-hidden'> (opens in new tab)</span></Anchor_>
```